app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -134,7 +139,7 @@ def transaction_reason_code
         end
 
         def reference
-          @reference ||= parsed_response_body.dig(:Status, :Reference)
+          @request_context&.dig(:settings, :Settings, :Reference)
```

Comment:
Get from the request context, other than from the response which we may not have it when vendor have 500 etc.

Reply:
Thanks for the explanation and I see why you did it this way. I'm not 100% sure we should look at the request (rather than the response) for everything, but approving as is.

Reply:
@charleyf , thanks for the feedback, It sounds a better idea to simply stuff. Remove extra info for now.